docs/source/cheat_sheet_py3.rst

```diff
@@ -334,19 +311,38 @@ Miscellaneous
        else:
            return sys.stdout
 
-   # Forward references are useful if you want to reference a class before
-   # it is defined
+
+You can even make your own duck types using :ref:`protocol-types`.
+
+Forward references
+******************
+
+.. code-block:: python
+
+   # You may want to reference a class before it is defined.
+   # This is known as a "forward reference".
    def f(foo: A) -> int:  # This will fail at runtime with 'A' is not defined
        ...
 
-   class A:
+   # However, if you add the following magic import:
```

Comment:
I don't think calling it magic is a good idea. I suggested a slightly different wording.
```suggestion
   # However, if you add the following special import:
```